config/routes.rb

```diff
@@ -301,6 +301,7 @@
       get '/activated' => 'idv#activated'
     end
     scope '/verify', module: 'idv', as: 'idv' do
+      get '/vendor_outage' => 'vendor_outage#show'
```

Comment:
Not in love with this route name - it's really more of a `gpo_only` warning page...

Reply:
Sure, but if PF is down, we're going to have to force users to gpo. My point is that either a vendor outage or an explicit decision for some other reason may trigger a GPO only flow.